ginkgo/run_command.go

```diff
@@ -5,14 +5,16 @@ import (
 	"fmt"
 	"math/rand"
 	"os"
+	"strings"
 	"time"
 
+	"io/ioutil"
```

Comment:
`goimports` doing it's thing. I'm happy to revert these lines if you'd prefer...